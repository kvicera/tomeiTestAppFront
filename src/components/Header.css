.Header {
  padding-top: 5em;
  min-width: 375px;
}
