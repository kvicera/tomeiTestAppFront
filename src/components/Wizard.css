.container {
  margin: 0 auto;
  padding-top: 2em;
}
/* Large screens */
@media all and (min-width: 800px) {
  .container {
    padding-left: 26%;
    padding-right: 26%;
  }
}

.Wizard {
  display: flex;
  flex-flow: row wrap;
  justify-content: space-around;
  position: relative;
}

.connecting-line {
  height: 2px;
  background: #006bce;
  position: absolute;
  width: 80%;
  margin: 0 auto;
  left: 0;
  right: 0;
  top: 3.5em;
  z-index: -5;
}
/* small screens */
@media all and (max-width: 360px) {
  .connecting-line {
    height: 2px;
    background: #006bce;
    position: absolute;
    width: 80%;
    margin: 0 auto;
    left: 0;
    right: 0;
    top: 2em;
    z-index: -5;
  }
}
@media all and (max-width: 450px) {
  .connecting-line {
    height: 2px;
    background: #006bce;
    position: absolute;
    width: 80%;
    margin: 0 auto;
    left: 0;
    right: 0;
    top: 2.7em;
    z-index: -5;
  }
}