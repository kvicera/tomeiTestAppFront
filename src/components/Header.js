import './Header.css'
import React from 'react'
import logo from '../assets/Logo.png'

function Header() {
  return (
    <div className="Header">
      <img src={logo} alt="logo"/>
    </div>
  )
}
export default Header
