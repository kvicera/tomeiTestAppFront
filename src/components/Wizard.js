import './Wizard.css'
import React from 'react'
// redux
// import { connect } from 'react-redux'
// import { startAction } from '../redux/actions/startAction'
// import { stopAction } from '../redux/actions/stopAction'

import WizardStep from './WizardStep'
import WizardForm from './WizardForm'

import step1Img from '../assets/Wizard-Step1.png'
import step2Img from '../assets/Wizard-Step2.png'
import step3Img from '../assets/Wizard-Step3.png'
import step4Img from '../assets/Wizard-Step4.png'
import step5Img from '../assets/Wizard-Step5.png'

function Wizard(props) {
  
  return (
    <div className="container">
      <div
        className="Wizard"
      >
        <div className="connecting-line"></div>
        <div className="BarBar"></div>
        <WizardStep
          activeStep={true}
          stepNum={1}
          stepDescription="CREATE YOUR ACCOUNT PASSWORD"
          stepImg={step1Img}
          alt="step1"
        />
        <WizardStep
          stepNum={2}
          stepDescription="PERSONAL INFORMATION"
          stepImg={step2Img}
          alt="step2"
        />
        <WizardStep
          stepNum={3}
          stepDescription="EMPLOYMENT DETAILS"
          stepImg={step3Img}
          alt="step3"
        />
        <WizardStep
          stepNum={4}
          stepDescription="UPLOAD DOCUMENTS"
          stepImg={step4Img}
          alt="step4"
        />
        <WizardStep
          stepNum={5}
          stepDescription="COMPLETE"
          stepImg={step5Img}
          alt="step5"
        />
      </div>
      <div>
        <WizardForm />
      </div>
    </div>
  )
}

// const mapStateToProps = (state) => ({
//   ...state,
// })

// const mapDispatchToProps = (dispatch) => ({
//   startAction: () => dispatch(startAction),
//   stopAction: () => dispatch(stopAction),
// })

// export default connect(mapStateToProps, mapDispatchToProps)(Wizard)
export default Wizard
