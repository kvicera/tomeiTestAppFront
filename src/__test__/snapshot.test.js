import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../components/Header'
import Wizard from '../components/Wizard'

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<Wizard />).toJSON()
  expect(tree).toMatchSnapshot()
})
