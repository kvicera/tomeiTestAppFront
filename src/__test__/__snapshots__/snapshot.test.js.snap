// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="Header"
>
  <img
    alt="logo"
    src="test-file-stub"
  />
</div>
`;

exports[`renders correctly when there are no items 2`] = `
<div
  className="container"
>
  <div
    className="Wizard"
  >
    <div
      className="connecting-line"
    />
    <div
      className="BarBar"
    />
    <div
      className="WizardStep"
    >
      <img
        alt="step1"
        className="StepImage"
        src="test-file-stub"
      />
      <div
        className="HideOnSmall"
      >
        <p
          className="StepLabelBoldActive"
        >
          STEP 
          1
          :
           
        </p>
        <p
          className="StepLabelSemiBoldActive"
        >
          CREATE YOUR ACCOUNT PASSWORD
        </p>
      </div>
    </div>
    <div
      className="WizardStep"
    >
      <img
        alt="step2"
        className="StepImage"
        src="test-file-stub"
      />
      <div
        className="HideOnSmall"
      >
        <p
          className="StepLabelBold"
        >
          STEP 
          2
          :
           
        </p>
        <p
          className="StepLabelSemiBold"
        >
          PERSONAL INFORMATION
        </p>
      </div>
    </div>
    <div
      className="WizardStep"
    >
      <img
        alt="step3"
        className="StepImage"
        src="test-file-stub"
      />
      <div
        className="HideOnSmall"
      >
        <p
          className="StepLabelBold"
        >
          STEP 
          3
          :
           
        </p>
        <p
          className="StepLabelSemiBold"
        >
          EMPLOYMENT DETAILS
        </p>
      </div>
    </div>
    <div
      className="WizardStep"
    >
      <img
        alt="step4"
        className="StepImage"
        src="test-file-stub"
      />
      <div
        className="HideOnSmall"
      >
        <p
          className="StepLabelBold"
        >
          STEP 
          4
          :
           
        </p>
        <p
          className="StepLabelSemiBold"
        >
          UPLOAD DOCUMENTS
        </p>
      </div>
    </div>
    <div
      className="WizardStep"
    >
      <img
        alt="step5"
        className="StepImage"
        src="test-file-stub"
      />
      <div
        className="HideOnSmall"
      >
        <p
          className="StepLabelBold"
        >
          STEP 
          5
          :
           
        </p>
        <p
          className="StepLabelSemiBold"
        >
          COMPLETE
        </p>
      </div>
    </div>
  </div>
  <div>
    <div
      className="WizardForm"
    >
      <div
        className="FormHeader"
      >
        <label
          className="FormHeaderTitle"
        >
          CREATE YOUR ACCOUNT
        </label>
      </div>
      <div
        className="FormPadding"
      >
        <div>
          <p
            className="FormDescription"
          >
            Because there will be documents that you need to prepare to apply for the loan, let's start off by creating a password so that you can login to your account once you have these document ready.
          </p>
        </div>
        <div>
          <div
            className="FormInputs"
          >
            <div
              className="AvatarUpload"
            >
              <img
                alt="avatar"
                className="AvatarPreview"
                src="test-file-stub"
              />
              <input
                className="AvatarInput"
                onChange={[Function]}
                type="file"
              />
              <button
                className="UploadButton"
                onClick={[Function]}
                type="file"
              >
                Upload
              </button>
            </div>
            <div
              className="FormInputsHolder"
            >
              <div
                className="FormRow1"
              >
                <div
                  className="Row"
                >
                  <label
                    className="FormLabel"
                  >
                    NAME
                  </label>
                  <input
                    className="FormInput"
                    onChange={[Function]}
                    type="text"
                  />
                </div>
                <div
                  className="Row"
                >
                  <label
                    className="FormLabel"
                  >
                    EMAIL
                  </label>
                  <input
                    className="FormInput"
                    onChange={[Function]}
                    type="text"
                  />
                </div>
                <div
                  className="Row"
                >
                  <label
                    className="FormLabel"
                  >
                    PASSWORD
                  </label>
                  <input
                    className="FormInput"
                    onChange={[Function]}
                    type="password"
                  />
                </div>
                <div
                  className="Row"
                >
                  <label
                    className="FormLabel"
                  >
                    CONFIRM PASSWORD
                  </label>
                  <input
                    className="FormInput"
                    onChange={[Function]}
                    type="password"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="FormFooter"
          >
            <button
              className="Button"
              onClick={[Function]}
            >
              <span
                className="SubmitBtnLabel"
              >
                SAVE & NEXT
              </span>
              <img
                alt="arrowBtn"
                className="SubmitBtnImg"
                src="test-file-stub"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
